kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/index.vue

```diff
@@ -59,7 +59,7 @@
             :disabled="availableRootCategories &&
               !availableRootCategories[key] &&
               !isKeyActive(key)"
-            iconAfter="chevronRight"
+            :iconAfter="hasNestedCategories(key) ? 'chevronRight' : null"
```

Comment:
You also have the `category` value available here which is the key for the `libraryCategories` object.